cocos/2d/assembler/label/text-processing.ts

```diff
@@ -78,6 +79,7 @@ export class TextProcessing {
         this._canvasData = CanvasPool.getInstance().get();
         this._canvas = this._canvasData.canvas;
         this._context = this._canvasData.context;
+        this._dpr = screenAdapter.devicePixelRatio || 1;
```

Comment:
devicePixelRatio is not correct, should calculate a ratio with screen.windowSize and canvas size